build.rs

```diff
@@ -472,7 +477,7 @@ fn build_c_code(
         })
         .collect::<Vec<_>>();
 
-    let test_srcs = RING_TEST_SRCS.iter().map(PathBuf::from).collect::<Vec<_>>();
+    let test_srcs = RING_TEST_SRCS.iter().map(|path| PathBuf::from(path)).collect::<Vec<_>>();
```

Comment:
I'm not sure why this change is made. I think it's not needed? The `clippy` job fails because of this.

Also, the `rustfmt` job fails, so I guess you need a `cargo fmt`.